python/lsst/ctrl/bps/bps_utils.py

```diff
@@ -150,3 +151,45 @@ def _create_execution_butler(config, qgraph_filename, execution_butler_dir, out_
         print(command, file=fh)
         print("\n", file=fh)  # Note: want a blank line
         subprocess.run(shlex.split(command), shell=False, check=True, stdout=fh, stderr=subprocess.STDOUT)
+
+
+def create_count_summary(counts):
+    """Create summary from count mapping.
+
+    Parameters
+    ----------
+    count : `collections.Counter` or `dict` [`str`, `int`]
+        Mapping of counts to keys.
+
+    Returns
+    -------
+    summary : `str`
+        Semi-colon delimited string of key:count pairs.
+        (e.g. "key1:cnt1;key2;cnt2")  Parsable by
+        parse_count_summary().
+    """
+    summary = ""
+    if counts is not None:
```

Comment:
I assume this check is here to make the function handle gracefully incorrect type of the input argument. In such a case, wouldn't it be better to just check if `counts` is an instance of a dict?